Civi/ActionSchedule/MappingInterface.php

```diff
@@ -23,9 +23,16 @@ interface MappingInterface {
   public function getId();
 
   /**
+   * Name of the table belonging to the main entity e.g. `civicrm_activity`
    * @return string
    */
-  public function getEntity();
+  public function getEntityTable(): string;
+
+  /**
+   * Main entity name e.g. `Activity`
+   * @return string
+   */
+  public function getEntityName(): string;
```

Comment:
Regarding `universe` - I don't see any published/contrib things that reference `MappingInterface`. So splitting this method (`getEntity()` => `getEntityTable()`+`getEntityName()`) should be OK.

Reply:
In my next commit i'm adding the `getEntity` function back in as a noisily-deprecated alias of `getEntityTable`. Just in case.